account_financial_report/tests/abstract_test.py

```diff
@@ -109,15 +108,7 @@ def _add_chart_of_accounts(self):
         })
         self.with_context(
             company_id=self.company.id, force_company=self.company.id)
-        wizard = self.env['wizard.multi.charts.accounts'].create({
```

Comment:
Not sure if a previous error in this method is still there in v12. See https://github.com/OCA/bank-payment/commit/9c5a98ffbe72531098b7c10c676b2c616284fd9b for details.

You can test the problem when having other chart template installed in the database doing the tests.

Any way, previous code is missing explicitly in the creation dictionary `'sale_tax_id' : False` and `'purchase_tax_id': False`, which avoids the problem.

Reply:
I don't understand, tests are green. How could another chart be present?

Reply:
When you are executing your tests in a customer environment CI (aka more or less integration tests) that has for example `l10n_es` installed.

Reply:
I installed `l10n_es` in a demo DB, executed tests of `account_financial_report` and no error was raised